packages/date-picker/src/picker.vue

```diff
@@ -389,6 +389,8 @@ export default {
     rangeSeparator: {
       default: '-'
     },
+    minimum: Date,
+    maximum: Date,
```

Comment:
New props that are actually set externally (using something like `<el-date-picker>`)